website/layouts/docs/baseof.html

```diff
@@ -13,7 +13,9 @@
           <div class="col-12 col-md-3 col-xl-2 td-sidebar d-print-none">
             {{ partial "sidebar.html" . }}
           </div>
-
+          <div class="d-none d-xl-block col-xl-2 td-toc d-print-none">
+            {{ partial "toc.html" . }}
```

Comment:
Did you forget to check this file in?

Reply:
@asmacdo this file is already in https://github.com/asmacdo/docsy/blob/827627c376a2f672a0b310ac7f3516d25bb87c33/layouts/partials/toc.html

Just a super non intuitive way hugo's decided to allow for overriding their [default](https://github.com/asmacdo/docsy/blob/827627c376a2f672a0b310ac7f3516d25bb87c33/layouts/docs/baseof.html)